### DISCUSSION TITLE
I made an upptime status page for the website

### DISCUSSION THREAD

**Discussion by markverb1** [FlagClicked/FlagClicked#65] [Show and tell]
I made a status page, powered by @upptime. It's just a replacement for the old uptimerobot one, because it is broken right now. (see #58) I know you want the URL to the page, so [click here](https://markverb1.github.io/upptime-test) to go to the page. Now this page just has to linked to status.whenflagclicked.org and the status page is complete.

**Comment by danielgallab** [comment]
I don't really want to use upptime because it lacks features and design, and the old one looked a lot better. I'll just use the APIs to create another one

**Reply by markverb1** [reply]
Okay, I guess we could use the upptime page while the custom one is being worked on.
